analysis/microbeMicrobeInteractions/tutorial_microbeMicrobeInteractions.m

```diff
@@ -13,21 +10,21 @@
 % tutorial can be adapted to any number of AGORA models and dietary conditions 
 % analyzed.
 % 
-% 
-% 
+initCobraToolbox(0);
+% path to where the COBRA Toolbox is located
+global CBTDIR
+cd([CBTDIR filesep 'tutorials' filesep 'analysis' filesep 'microbeMicrobeInteractions'])
 % We will use the AGORA resource (Magnusdottir et al., Nat Biotechnol. 2017 
-% Jan;35(1):81-89) in this tutorial. Please download AGORA version 1.02  from 
-% <https://vmh.life https://vmh.life> and place the models into a folder.
-% 
-% Define the path to the folder where you stored the AGORA models.
-
-modelPath='YOUR_PATH_TO_AGORA/';
+% Jan;35(1):81-89) in this tutorial. AGORA version 1.02 is available at
+% www.vmh.life. Download AGORA and place the models into a folder.
+system('curl -O https://www.vmh.life/files/reconstructions/AGORA/1.02/Agora-1.02.zip')
+unzip('Agora-1.02.zip','AGORA')
+modPath=[CBTDIR filesep 'tutorials' filesep 'analysis' filesep 'microbeMicrobeInteractions' filesep 'AGORA' filesep 'mat'];
```

Comment:
if you use the `fileparts/which` command from above, this would then be:
`modPath = [tutorialPath filesep 'AGORA' filesep 'mat'];`